.github/bot-files/comments.yml

```diff
@@ -2,7 +2,7 @@
 # for now you'll need to difine every comment here, in the future comments will be optional.
 # please see the example for the file below
 
-prFirstTimeContributor: Welcome to the Kogito Community!. Please make sure you've read the [contributors' guide](https://github.com/kiegroup/kogito-cloud-operator/blob/master/README.md#contributing-to-the-kogito-operator)
+prFirstTimeContributor: Welcome to the RHPAM Kogito Community!. Please make sure you've read the [contributors' guide](https://github.com/kiegroup/kogito-cloud-operator/blob/master/README.md#contributing-to-the-kogito-operator)
```

Comment:
```suggestion
prFirstTimeContributor: Welcome to the RHPAM Kogito Community!. Please make sure you've read the [contributors' guide](https://github.com/kiegroup/kogito-operator/blob/master/README.md#contributing-to-the-kogito-operator)
```

Reply:
I waiting for rename PR to get merge to rename `kogito-cloud-operator` word with `kogito-operator`. I am adding RHPAM in community title

Reply:
done